README.md

```diff
@@ -75,25 +75,29 @@ $ npm start
 
 Generate a Github personal access token using these [Instructions](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) `Make sure you don't select any scope just generate a simple token`
 
-Copy the token and open Chrome Developer Console to convert your token to base64 so github do not revert your token when you push your token to git
+1. Create a file called .env in the root of your project's directory.
```

Comment:
Change the sentence as `root directory of your project` instead of `root of your project directory`.